### ISSUE TITLE
Embedding layer in bidirectional LSTM

### ISSUE THREAD

**Issue by kvarada** [tflearn/tflearn#847]
Thank you for making this library available. 

I want to verify whether I am using my bidirectional LSTM with the embedding layer correctly or not. 
Below I am pasting my code for network building and initializing the embedding weights with glove embeddings. Since my LSTM input dimensions are 128, I am getting the first 128 elements of the glove embedding vector. 

Can you please let me know whether I am doing the right thing? 

`import tflearn
import sys, os, argparse, pickle, random, smart_open
import tensorflow as tf
from tflearn.data_utils import to_categorical, pad_sequences
from tflearn.datasets import imdb
from tflearn.layers.core import input_data, dropout, fully_connected
from tflearn.layers.embedding_ops import embedding
from tflearn.layers.recurrent import bidirectional_rnn, BasicLSTMCell
from tflearn.layers.estimator import regression
import numpy as np

def build_network(ilearning_rate=0.001):
    # Network building
    net = input_data(shape=[None, 200])
    net = embedding(net, input_dim=51887, output_dim=128, trainable=False, name="EmbeddingLayer")
    net = bidirectional_rnn(net, BasicLSTMCell(128), BasicLSTMCell(128))
    net = dropout(net, 0.5)
    net = fully_connected(net, 2, activation='softmax')

    net = regression(net, optimizer='adam', loss='categorical_crossentropy'
                     , learning_rate=ilearning_rate
                     )

    return net

def train(net, trainX, trainY, model_path, ibatch_size=512):
    # Training
    model = tflearn.DNN(net, clip_gradients=0., tensorboard_verbose=0)
    # Retrieve embedding layer weights (only a single weight matrix, so index is 0)
    embeddingWeights = tflearn.get_layer_variables_by_name('EmbeddingLayer')[0]
    print('default embeddings: ', embeddingWeights[0])
    model.set_weights(embeddingWeights, glove_embeddings)
    model.fit(trainX, trainY, validation_set=0.1, n_epoch=10, show_metric=True, batch_size=ibatch_size)
    return model`
